src/main/java/com/dev/cinema/Main.java

```diff
@@ -167,5 +170,51 @@ public static void main(String[] args) {
 
         shoppingCartService.clear(cartByUser3);
         System.out.println(cartByUser3);
+
+        System.out.println("+++++ORDER SERVICE+++++");
+
+        final OrderService orderService = (OrderService) injector
+                .getInstance(OrderService.class);
+
+        Ticket additionalTicket1 = new Ticket();
+        Ticket additionalTicket2 = new Ticket();
+        Ticket additionalTicket3 = new Ticket();
+        Ticket additionalTicket4 = new Ticket();
+        Ticket additionalTicket5 = new Ticket();
+        Ticket additionalTicket6 = new Ticket();
+
+        for (Ticket ticket : Arrays.asList(additionalTicket1, additionalTicket2,
+                additionalTicket3)) {
+            ticket.setMovieSession(movieSessionForToday);
+        }
+        for (Ticket ticket : Arrays.asList(additionalTicket4, additionalTicket5,
+                additionalTicket6)) {
+            ticket.setMovieSession(movieSessionForTomorrow);
+        }
+        for (Ticket ticket4 : Arrays.asList(additionalTicket1, additionalTicket2,
+                additionalTicket3)) {
+            ticket4.setUser(registeredMykola);
+        }
```

Comment:
what is the purpose of this?

Reply:
I don't know)I have been building the ordering flow and it got where I got lost.
Removed, thank you